src/group_theory/quotient_group.lean

```diff
@@ -51,6 +51,15 @@ instance : is_group_hom (mk : G → quotient N) := ⟨λ _ _, rfl⟩
 attribute [to_additive quotient_add_group.is_add_group_hom] quotient_group.is_group_hom
 attribute [to_additive quotient_add_group.is_add_group_hom.equations._eqn_1] quotient_group.is_group_hom.equations._eqn_1
 
+@[simp] lemma quotient_group.ker :
```

Comment:
This also needs a `to_additive` attribute.

Reply:
Hopefully I got it right.